swagger/v2/defence_organisation.json

```diff
@@ -15,9 +15,9 @@
     "type": {
       "description": "The defence_organisations type",
       "enum": [
-        "defence_organisations"
+        "defence_organisation"
```

Comment:
Is this not plural as in the comment from hearings that you discussed with me:

```
"type": {
      "description": "The hearing type",
      "example": "hearings",
      "type": "string"
    },
```

Reply:
Good spot - will correct

Reply:
I've sorted this by referencing a new "included_resource" object within the same file.  In the payload, it is pluralised when in the "included" but not when listed as a relationship. Having a separate object accommodates both scenarios.

Reply:
👍

Reply:
I wouldn't worry too much about defence organisation. We need to stop offering this because it is coming from CDA data, not Common Platform. So it's likely we'll be removing the option from the api - needs discussing further